v2/fleet-local/control/flight-director@.service

```diff
@@ -47,6 +45,8 @@ ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
   -e GITHUB_CLIENT_ID=`etcdctl get /FD/GITHUB_CLIENT_ID` \
   -e GITHUB_CLIENT_SECRET=`etcdctl get /FD/GITHUB_CLIENT_SECRET` \
   -e AUTHORIZER_TYPE=`etcdctl get /FD/AUTHORIZER_TYPE` \
+  -e FD_MARATHON_MASTER_PROTOCOL=`etcdctl get /FD/FD_MARATHON_MASTER_PROTOCOL` \
+  -e FD_ALLOW_MARATHON_UNVERIFIED_TLS=`etcdctl get /FD/FD_ALLOW_MARATHON_UNVERIFIED_TLS` \
```

Comment:
It's configurable (off by default). Accessing the internal Mesos clusters API requires this to be set to true due to an untrusted cert:  https://www.dropbox.com/s/vyrt2xyxev41u2k/Screenshot%202016-03-07%2010.30.54.png?dl=0 .